mne/time_frequency/tfr.py

```diff
@@ -29,52 +30,73 @@
 from ..externals.six import string_types
 
 
-def _get_data(inst, return_itc):
-    """Get data from Epochs or Evoked instance as epochs x ch x time"""
-    from ..epochs import _BaseEpochs
-    from ..evoked import Evoked
-    if not isinstance(inst, (_BaseEpochs, Evoked)):
-        raise TypeError('inst must be Epochs or Evoked')
-    if isinstance(inst, _BaseEpochs):
-        data = inst.get_data()
-    else:
-        if return_itc:
-            raise ValueError('return_itc must be False for evoked data')
-        data = inst.data[np.newaxis, ...].copy()
-    return data
+# Make wavelet
+
+@deprecated("This function will be removed in mne 0.14.")
+def morlet(sfreq, freqs, n_cycles=7.0, sigma=None, zero_mean=False):
```

Comment:
> It looks like more something relevant to scipy.signal no?

Probably, but the development timeline there is longer, and I doubt we have anyone willing to do the PR. I'd say leave the existing one alone -- even though it's inconsistent with DPSS, it has been that way a long time without creating problems AFAIK